docs/admin/monitoring/export-logs.md

```diff
@@ -0,0 +1,139 @@
+Copy `/srv/hops/logstash/config/services.conf` to `/srv/hops/logstash/config/services_http.conf`
+
+Change the pipeline *input address* to:
+```treetop
+input {
+  pipeline {
+    address => services_http
+  }
+}
+```
+
+!!! note
+    Take a note of the pipeline address as we will use it in Step 2
+
+At the end of the file is the `output` section which currently forwards them to OpenSearch. Replace the output section with a sample block such as
+
+```treetop
+output {
+  http {
+    format => "json_batch"
+    headers => ["x-api-key", "API_KEY"]
+    http_compression => false
+    http_method => "post"
+    url => "https://localhost/logs"
+    follow_redirects => false
+  }
+}
+```
+
+#### Step 2
+The next step is to configure Logstash to use the new pipeline.
+
+Open `/srv/hops/logstash/config/pipelines.yml`
+
+Add the new pipeline in the pipeline definitions
+```yaml
+- pipeline.id: services_http
+  path.config: "/srv/hops/logstash/config/services_http.conf"
+  pipeline.batch.delay: 2000
+  pipeline.batch.size: 50
+```
+
+**Instruct** the services pipeline to push logs also in the newly created pipeline by appending to `services-intake` for example:
+
+```yaml
+- pipeline.id: services-intake
+  config.string: |
+    input { beats { port => 5053 } }
+    output { pipeline { send_to => ["services","services_http"] } }
+```
+
+#### Step 3
+Final step is to restart Logstash with `sudo systemctl restart logstash`
+
+Logstash logs can be found in `/srv/hops/logstash/log/logstash-plain.log`
+
+
+### Export Spark logs
+To stream applications' logs to another system the Steps are fairly similar to exporting services logs but need some additional configuration.
+
+#### Step 1
+Copy `/srv/hops/logstash/config/spark-streaming.conf` to `/srv/hops/logstash/config/spark-streaming_http.conf`
+
+Change the **input** section to:
+
+```treetop
+input {
+  pipeline {
+    address => spark_http
+  }
+}
+```
+
+Also, add the **output** block such as:
+
+```treetop
+output {
+  http {
+    format => "json_batch"
+    headers => ["x-api-key", "API_KEY"]
+    http_compression => false
+    http_method => "post"
+    url => "https://localhost/logs"
+    follow_redirects => false
+  }
+}
+```
+
+#### Step 2
+Edit `/srv/hops/logstash/config/spark-streaming.conf` and **change** the input to:
+
+```treetop
+input {
+  pipeline {
+    address => spark
+  }
+}
+```
+
+#### Step 3
+Now you need to change `/srv/hops/logstash/config/pipelines.yml` and **add** the following pipeline definitions:
+
+```yaml
+- pipeline.id: spark-intake
+  config.string: |
+    input { beats { port => 5044 } }
+    output { pipeline { send_to => ["spark", "spark_http"] } }
+- pipeline.id: spark_http
+  path.config: "/srv/hops/logstash/config/spark-streaming_http.conf"
+```
+
+#### Step 4
+Finally you should restart Logstash `sudo systemctl restart logstash`
+
+## Conclusion
+It is not easy to write a guide for a task that can be achieved in many different ways but in this guide we gave solid
```

Comment:
Users don't care that it's not easy. Just refer them to the different plugins configuration to understand how they need to configure their new pipeline to send data wherever they need to.